### PR TITLE
XFT2-1295 add radius endpoint for suggested radius api

### PR DESCRIPTION
**Pull request by chloext**

### CODE REVIEW

File: predicthq/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "2.1.1"
+__version__ = "2.1.2"
```
Comment (CockyAmoeba) [inline predicthq/version.py:1]: Should this not be a minor version bump at least as it is the addition of a new feature - not a bug fix or improvement?

File: usecases/suggested_radius.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ACCESS_TOKEN = 'abc123'
+
+phq = Client(access_token=ACCESS_TOKEN)
+
+
+suggested_radius = phq.radius.search(location__origin="45.5051,-122.6750")
+print(suggested_radius)
+
+
+suggested_radius = phq.radius.search(location__origin="45.5051,-122.6750", radius_unit="mi", industry="retail")
```
Comment (CockyAmoeba) [inline usecases/suggested_radius.py:15]: perhaps mention the other industries or link through to relevant documentation

Comment (chloext) [reply usecases/suggested_radius.py:18]: updated

File: predicthq/version.py (hunk 1 above)
Comment (chloext) [reply predicthq/version.py]: updated

**Review by CockyAmoeba — APPROVED** [review]

**Pull request merged**